libraries/layouts/Antispam_field_layout.php

```diff
@@ -44,7 +44,7 @@ public function fields()
 		$fields['akismet_api_key'] = array('class' => 'akismet_api_key', 'label' => 'Akismet API key');
 		$fields['recaptcha_public_key'] = array('class' => 'recaptcha_public_key', 'label' => 'reCAPTCHA public key', 'size' => 60);
 		$fields['recaptcha_private_key'] = array('class' => 'recaptcha_private_key', 'label' => 'reCAPTCHA private key', 'size' => 60);
-		$fields['recaptcha_theme'] = array('type' => 'select',  'label' => 'reCAPTCHA theme', 'options' => array('clean' => 'clean', 'blackglass' => 'blackglass', 'white' => 'white', 'red' => 'red'), 'class' => 'recaptcha_theme');
+		$fields['recaptcha_theme'] = array('type' => 'select',  'label' => 'reCAPTCHA theme', 'options' => array('light' => 'light', 'dark' => 'dark'), 'class' => 'recaptcha_theme');
```

Comment:
Themes are from https://developers.google.com/recaptcha/docs/faq#can-i-customize-the-recaptcha-widget

Reply:
I'd include that link in the comments for reference.